tests/playTests.sh

```diff
@@ -322,6 +323,19 @@ ZCAT=./zstdcat "$ZSTDGREP" 2>&1 "1234" tmp_grep_bad.zst && die "Should have fail
 ZCAT=./zstdcat "$ZSTDGREP" 2>&1 "1234" tmp_grep_bad.zst | grep "No such file or directory" || true
 rm -f tmp_grep*
 
+println "\n===> zstdless tests"
+ln -sf "$ZSTD_BIN" zstd
+rm -f tmp_less*
+echo "1234" > tmp_less
+zstd -f tmp_less
+lines=$(ZSTD=./zstd "$ZSTDLESS" 2>&1 tmp_less.zst | wc -l)
+test 1 -eq $lines
+ZSTD=./zstd "$ZSTDLESS" -f tmp_less.zst > tmp_less_regenerated
+cat tmp_less_regenerated
+$DIFF tmp_less tmp_less_regenerated
+ZSTD=./zstd "$ZSTDLESS" 2>&1 tmp_less_bad.zst | grep "No such file or directory" || true
```

Comment:
Shouldn't this be `|| die "..."`? You're trying to assert that you got this error message, right?